src/BodyPipe.cc

```diff
@@ -133,12 +133,12 @@ BodyPipe::BodyPipe(Producer *aProducer): theBodySize(-1),
     // TODO: teach MemBuf to start with zero minSize
     // TODO: limit maxSize by theBodySize, when known?
     theBuf.init(2*1024, MaxCapacity);
-    debugs(91,7, "created BodyPipe" << status());
+    debugs(91,7, "created BodyPipe " << status());
 }
 
 BodyPipe::~BodyPipe()
 {
-    debugs(91,7, "destroying BodyPipe" << status());
+    debugs(91,7, "destroying BodyPipe " << status());
```

Comment:
Please take advantage of the touch to make this produce the log pattern detected by the `scripts/find-alive.pl`:
```suggestion
    debugs(91,7, "destruct, this=" << static_cast<void*>(this) << ", " << status());
```